examples/simple_http_oauth_server.py

```
@@ -0,0 +1,539 @@
+        <head>
+            <title>Error {status_code}</title>
+            <style>
+                body {{ font-family: Arial, sans-serif; max-width: 800px; margin: 50px auto; padding: 20px; }}
+                .error {{ background: #f8d7da; color: #721c24; padding: 15px; border-radius: 5px; margin: 20px 0; }}
+                .button {{ display: inline-block; padding: 10px 20px; margin: 10px; background: #007bff; color: white; text-decoration: none; border-radius: 5px; }}
+            </style>
+        </head>
+        <body>
+            <h1>❌ Error {status_code}</h1>
+            <div class="error">
+                <p>{html.escape(message)}</p>
```

Ruff (0.13.1)

Local variable `html` referenced before assignment

(F823)